### ISSUE TITLE
Expose path to download database for `downloadDB` as a user input

### ISSUE THREAD

**Issue by valentynbez** [motu-tool/mOTUs#109]
Databases and software are usually stored in different folder in the filesystem for various reasons. The choice of path for db download should be up to user.

**Comment by AlessioMilanese** [comment]
At the moment you can specify the path of the database with the oprion `-db`. I think that is what you are looking for?

**Comment by valentynbez** [comment]
No, I mean the ability to change the download location for the database in `downloadDB.py`.

**Comment by AlessioMilanese** [comment]
We can add it as an advance option. But it might make it more complicated for non-advanced users.
Also, if you downoad in a different position, then you will need to specify `-db` when running `motus`.

**Comment by jakob-wirbel** [comment]
I would like to plus 1 that feature request 😄 
It would be super helpful if you could download the database to a specific folder (maybe somewhere else in the filesystem), and yes, you would have to specify then the path to the `db` anytime you run `motus`, but that should be fine

**Comment by donovan-h-parks** [comment]
I would also like to plus 1 this feature request.

**Comment by hjruscheweyh** [comment]
Noted. Will be added in mOTUsv4
- If no destination path is set --> store at default location
- If destination path is set --> store there, requires user to remember/set this path

**Comment by hindrek** [comment]
> Noted. Will be added in mOTUsv4
> 
> * If no destination path is set --> store at default location
> * If destination path is set --> store there, requires user to remember/set this path

Hi! Has there been any progress with the matter?

**Comment by vivekbibs** [comment]
> > Noted. Will be added in mOTUsv4
> > 
> > * If no destination path is set --> store at default location
> > * If destination path is set --> store there, requires user to remember/set this path
> Hi! Has there been any progress with the matter?

I Also would like to know about any advances with this matter. It seems to me that there is still no --db option so specifiy the database path.
Thanks in advance !

**Comment by jorondo1** [comment]
I second this! Alternatively, is there a way to manually download the database (using a link) ?

**Comment by hjruscheweyh** [comment]
Hi Everyone

We decided to push this feature to the next release (4.1) as mOTUs 4.0 is a complete reimplementation of mOTUs3 and we aimed it to be as stable as possible.

marker gene and annotation databases are at Zenodo which, for unknown reason, is very slow for downloading files these days.  

https://zenodo.org/records/17668622
https://zenodo.org/records/17669279

**Comment by jorondo1** [comment]
I understand. Is it currently possible to point to a downloaded database? -db or --db are not recognize, I don't see anything in `motus profile -h`
